tests/fracs/test_plane_fracture.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+import pytest
+
+from porepy.fracs import plane_fracture
+
+# Test computation of fracture centroids.
+
+
+def test_frac_3d_1():
```

Comment:
Parametrization of this set of tests, please.

Reply:
Agreed, doing it